packages/rs-drive-abci/src/execution/types/execution_event/mod.rs

```diff
@@ -25,6 +28,7 @@ pub(in crate::execution) enum ExecutionEvent<'a> {
         identity: PartialIdentity,
         /// the operations that the identity is requesting to perform
         operations: Vec<DriveOperation<'a>>,
+        signature_verifications: Option<Vec<SignatureVerificationOperation>>,
```

Comment:
validation_costs

Reply:
They should not be an Option.

Reply:
This should contain a vec of validation Operations, which should be an enum

Reply:
Fixed